src/endout/endout.h

```diff
@@ -2,19 +2,130 @@
 #include "generalTools/constants.h"
 #include "range/v3/all.hpp"
 #include "generalTools/sigfig.h"
-//#include "generalTools/tools.h"
 #include "coherentElastic/coherentElastic.h"
 #include "endout/endout_util/writeInelasticToENDF.h"
 #include "endout/endout_util/writeIncElasticToENDF.h"
 #include "endout/endout_util/writeCohElasticToENDF.h"
 #include "ENDFtk.hpp"
+#include "lipservice.hpp"
 
 using namespace njoy::ENDFtk;
 using Elastic         = section::Type<7,2>;
 using CoherentElastic = section::Type<7,2>::CoherentElastic;
 using Inelastic       = section::Type<7,4>;
 using ScatteringLawConstants = section::Type<7,4>::ScatteringLawConstants;
 
+template <typename Range> 
+auto scaleDebyeWallerCoefficients( const nlohmann::json& jsonInput,
```

Comment:
It might be better to just pass the parts of the jsonInput that you want instead of everything. It's a matter of preference.

Reply:
I assumed it would be cleaner to just pass the whole json input into whatever function it needs, so you don't have to slice out the relevant pieces. I understand why you might prefer it and if you want me to change it I of course can, but if it's up to me I find it cleaner to just keep it simple.